docs/vocascan-server/run-programmatically.md

```diff
@@ -0,0 +1,50 @@
+# Run programmatically
+
+We also provide a way to integrate the vocascan-server directly into your nodeJS application. This could be useful if
+you want to run vocascan-server from inside of your electron client.
+
+## 1. Install via npm
+
+We publish packages to [npm](https://npmjs.com/package/@vocascan/server) and
+[GitHub Package Registry](https://github.com/vocascan/vocascan-server/packages/1077993).
+
+```bash
+npm i @vocascan/server
+```
+
+## 2. Run
+
+You can then import the `@vocascan/server` package and run the server. The first argument is an object containing your
```

Comment:
be more direct, specifically with the beginnings of sentences.: say Import[...]  instead of You can  /  You Could  etc.
This applies throughout the documentation.

so change the first phrase to Import the[....]